examples/demo2/tabs/about.rs

```diff
@@ -0,0 +1,153 @@
+use itertools::Itertools;
+use ratatui::{prelude::*, widgets::*};
+
+use crate::{colors, layout, THEME};
+
+const RATATUI_LOGO: [&str; 32] = [
+    "                                ",
+    "              ████             ",
+    "            ███████             ",
+    "           ████████             ",
+    "          █████████             ",
+    "         ██████████             ",
+    "        ████████████            ",
+    "        █████████████           ",
+    "        █████████████     ██████",
+    "         ███████████    ████████",
+    "              █████ ███████████ ",
+    "               ███ ██ee████████ ",
+    "                █ █████████████ ",
+    "            ████ █████████████  ",
+    "           █████████████████    ",
+    "           ████████████████     ",
+    "           ████████████████     ",
+    "            ███ ██████████      ",
+    "          ██    █████████       ",
+    "         █xx█   █████████       ",
+    "        █xxxx█ ██████████       ",
+    "       █xx█xxx█ █████████       ",
+    "      █xx██xxxx█ ████████       ",
+    "     █xxxxxxxxxx█ ██████████    ",
+    "    █xxxxxxxxxxxx█ ██████████   ",
+    "   █xxxxxxx██xxxxx█ █████████   ",
+    "  █xxxxxxxxx██xxxxx█ ████  ███  ",
+    " █xxxxxxxxxxxxxxxxxx█ ██   ███  ",
+    "█xxxxxxxxxxxxxxxxxxxx█ █   ███  ",
+    " █xxxxxxxxxxxxxxxxxxxx█   ███   ",
+    "  █xxxxxxxxxxxxxxxxxxxx█ ██     ",
+    "   █xxxxxxxxxxxxxxxxxxxx█ █     ",
+];
```

Comment:
I generated this by taking the original Ratatui artwork (at 200x200 or so) and scaled it down and then manually pixel edited every pixel by hand to make it seem decent. Exported and then manually converted with a bunch of find and replace. Any errors in making this look like the original image are mine (though @kdheepak helped out with the hat and eyes).